Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -857,6 +856,8 @@ extension CarPlayManager: CarPlayNavigationViewControllerDelegate {
         if let passiveLocationProvider = navigationMapView?.mapView.location.locationProvider as? PassiveLocationProvider {
             passiveLocationProvider.locationManager.resumeTripSession()
         }
+        
+        self.carPlayNavigationViewController = nil
```

Comment:
It's to nil out the `CarPlayNavigationViewController` in `CarPlayManager` when the `CarPlayNavigationViewController` dismissed and exit navigation